pkg/autoupdate/tuf/autoupdate.go

```diff
@@ -123,6 +137,12 @@ func initMetadataClient(rootDirectory, metadataUrl string, metadataHttpClient *h
 		return nil, fmt.Errorf("could not make local TUF directory %s: %w", localTufDirectory, err)
 	}
 
+	// Ensure that directory permissions are correct, otherwise TUF will fail to initialize. We cannot
+	// have permissions in excess of -rwxr-x---.
+	if err := os.Chmod(localTufDirectory, 0750); err != nil {
```

Comment:
IIRC this has no effect on windows.

Reply:
I think that's fine, but let me check!

Reply:
Yup: https://github.com/theupdateframework/go-tuf/blob/master/internal/fsutil/perm_windows.go#L7